src/locales/de.json

```
@@ -88,5 +90,9 @@
 
   "wgu-localeswitcher": {
     "title": "Spracheinstellungen"
+  },
+
+  "wgu-themeswitcher": {
+    "title": "Dunklen Modus"
```

```suggestion
    "title": "Dunkler Modus"
```

A minor linguistic correction